buddyboss/buddyboss-platform
# PROD-693 #3140 - Fix: Activity post auto-refresh in activity page even if activity auto-refresh setting disabled

https://buddyboss.atlassian.net/browse/PROD-693

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3140 .

### How to test the changes in this Pull Request:

1. Go to BB > Settings > Activity > Activity auto-refresh and disable it - https://snipboard.io/tcTDah.jpg
2. Go to on frontend activity page
3. Open the new activity page on a different browser with a different user
4. Try to post something on an activity page
5. Check other browser activity page
6. You can see it will not auto refresh even if it is disabled in the settings.

### Proof Screenshots or Video
https://vimeo.com/646766367/c4ee91f6b7


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
